### DISCUSSION TITLE
Update Feed/Subscription

### DISCUSSION THREAD

**Discussion by zkxs** [zkxs/neos-mod-list#9] [Ideas]
We need some way of keeping users updated regarding new versions of mods. I don't think simply showing the last date updated on a page is enough, as that would require users to actively check if their mods are outdated which they can't be trusted to do reliably.

Here's my ideas:
- Build an autoupdater into NeosModLoader itself. I don't like this idea for security reasons.
- Convince users to make github accounts, star the mods they use, then check the github home page periodically. This is awful.
- Create a #mod-updates discord channel somewhere with a role where users can opt in to pings. This would ping for *every* mod update.
  - Creating a role per mod might be viable?
- Create a discord channel per-mod and let users right click to opt into pings on their own, because screw making a role for every mod update channel. This would involve some nightmare server with a zillion nearly empty channels people would have to scroll through.
- Create some sort of bot that lets you subscribe to specific mods. The bot would then notify users when there's an update.
  - Mass discord DMs won't work because discord bots suck
  - Email bots would probably cost money at scale

All of these options suck. Which one do you think sucks the least? Do you have any ideas that don't suck?

Of what's here currently, I think the #mod-updates discord channel with a role per mod is the least bad option. It would need a bot to let users self-assign roles.

**Comment by EIA485** [comment]
for now i think a discord server with per mod roles is the best option. 
in the long term a mod loader gui that runs every time you start neos seems like the best option to me.

**Reply by art0007i** [reply]
Mod loader gui shouldn't be too difficult especially because we can look at how others did it already
https://github.com/DeadlyKitten/MonkeModManager
https://github.com/Assistant/ModAssistant

**Reply by ghost** [reply]
> for now i think a discord server with per mod roles is the best option. in the long term a mod loader gui that runs every time you start neos seems like the best option to me.

Where is the discord i want to be in it

**Reply by zkxs** [reply]
> Where is the discord i want to be in it

There is no Discord server yet. I've been waiting to see if the Neos team provides more detailed modding rules before making the Discord, so that I can make sure the Discord server's rules match the Neos rules. When I do end up creating a Discord server, I'll link to it from the appropriate READMEs in my GitHub projects.
